### PR TITLE
feat: Created basic introduction.html

### PR DESCRIPTION
**Pull request by flashattention**
Set document type to HTML5
Set charset to "UTF-8"
Set title to "My introduction"
Set H1 content to "Eunbae Jeon"
Set P content to a short introduction

### CODE REVIEW

File: introduction.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<html>
+<head>
+ <meta charset="UTF-8">
+ <title>My introduction</title>
+</head>
+<body>
+ <h1>Eunbae Jeon</h1>
+ <p>Hi, my name is Eunbae Jeon. I'm a student at Fastcampus studying front-end engineering.</p>
```
Comment (dozukwang) [inline introduction.html]: could you divide this paragraph into 2 paragraphs?

File: introduction.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<html>
```
Comment (dozukwang) [inline introduction.html:2]: please close <html> tag

**Review by dozukwang — CHANGES_REQUESTED** [review]
please check my reviews.

File: introduction.html (hunk 1 above)
Comment (flashattention) [reply introduction.html]: Got it.

File: introduction.html (hunk 2 above)
Comment (flashattention) [reply introduction.html:2]: Done!

**Review by dozukwang — APPROVED** [review]
all codes work fine ^^.

**Pull request merged**